src/index.tsx

```diff
@@ -0,0 +1,3 @@
+import Img from './react-image'
+
+export default Img
```

Comment:
i think a succinct way to do this is
(just re-commenting with multiline suggestion)

```suggestion
export {default} from './react-image'
```

Reply:
Rollup says:

```./src/index.tsx → cjs...
(!) Mixing named and default exports
https://rollupjs.org/guide/en/#output-exports
The following entry modules are using named and default exports together:
src/index.tsx

Consumers of your bundle will have to use chunk['default'] to access their default export, which may not be what you want. Use `output.exports: 'named'` to disable this warning
created cjs in 2.8s
```
Which is why I created this file in the first place. Suggestions?

Reply:
you can just use one style then

```suggestion
export {default as Image} from './react-image'
export {default as useImage, useImageProps} from './useImage'
```

the usage would be:

```js
import {Image, useImage} from 'react-image';
```

can see if that works

Reply:
That would break backward compatibility with users that are used to doing `import Img from 'react-image'`. Perhaps I should just update the examples for hooks to `import useImage from 'react-image/useImage'` ?

Reply:
yes that is better.
helps with tree-shaking too 👍